src/System.Diagnostics.FileVersionInfo/tests/NativeConsoleApp/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
That is what I was asking with the first question :)

I just double checked and we don't build *.vcxproj so your right the build won't pick these up unless there is a ProjectReference to them. That is what I want.